### PR TITLE
zoro url fix

### PR DESCRIPTION
**Pull request by Th3-A6add0n**
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/en/zoro/build.gradle [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ext {
-    extName = 'AniWatch'
-    extClass = '.AniWatch'
+    extName = 'HiAnime'
+    extClass = '.HiAnime'
     themePkg = 'zorotheme'
-    baseUrl = 'https://aniwatch.to'
+    baseUrl = 'https://hianime.to/'
```
Comment (Secozzi) [inline src/en/zoro/build.gradle:5]: Remove trailing `/`

File: src/en/zoro/res/mipmap-hdpi/ic_launcher.png
Comment (Secozzi) [inline src/en/zoro/res/mipmap-hdpi/ic_launcher.png:1]: Create square icons using the tool in `CONTRIBUTING.md`

**Comment by Secozzi** [comment]
We appreciate pull requests, but the changes needs to be built and tested using android studio. I can tell that the changes weren't actually tested, so I'm marking as draft until you can show that you built and tested the extension.

**Comment by Th3-A6add0n** [comment]
> We appreciate pull requests, but the changes needs to be built and tested using android studio. I can tell that the changes weren't actually tested, so I'm marking as draft until you can show that you built and tested the extension.

I'll test this first thing in the morning.

**Comment by Secozzi** [comment]
Follow the checklist as well, and for next time create a separate branch instead of making a pull request from master.

File: lib-multisrc/zorotheme/build.gradle.kts [hunk 2]
```diff
@@ -2,7 +2,7 @@ plugins {
     id("lib-multisrc")
 }
 
-baseVersionCode = 1
+baseVersionCode = 2
```
Comment (Secozzi) [inline lib-multisrc/zorotheme/build.gradle.kts:5]: This should only be bumped on a theme change, (and you shouldn't bump both this and `overrideVersionCode`, only bump `overrideVersionCode`.

**Pull request closed without merging**